app.rb

```diff
@@ -0,0 +1,108 @@
+require './person'
+require './book'
+require './rental'
+require './teacher'
+require './student'
+require 'date'
+
+class App
+  attr_accessor :people, :books, :rentals
+
+  def initialize
+    @people = []
+    @books = []
+    @rentals = []
+  end
+
+  def add_person
+    print 'Do you whant to create a Student (1) 0r Teacher (2)? [input the number]: '
+    job = gets.chomp.to_i
+    case job
+    when 1
+      print 'name: '
+      name = gets.chomp
+      print 'Age: '
+      age = gets.chomp
+      print 'Has parent permissiom [Y/N]? '
```

Comment:
- [ ] I think it will be better if you include a function that will handle the wrong input for the parent's permission. When I was trying to add a person, I entered the letter 'z' for parent permission and the person still got created which might cause some errors in the UI later on 👍🏽 .

It will also be better if you write a function that will return false if parent permission is 'n' and true if parent permission is 'y' 😄

Reply:
# Thank you @vickymar for reviewing my project.
For sure, I had to validate this user input to avoid errors,
in the following commit, I have added a switch case to validate the user input to avoid errors.
![image](https://user-images.githubusercontent.com/102757126/194688623-5ef0b618-c7a5-4fe6-8389-9df429adf836.png)

Thanks!